spec/person_spec.rb

```diff
@@ -0,0 +1,8 @@
+require './person'
+describe Person do
+  let(:person) { Person.new(31, 'Mumbere Habert') }
+  it 'should initialize correctly' do
+    expect(person.age).to eq(31)
+    expect(person.name).to eq('Mumbere Habert')
+  end
```

Comment:
- Kindly, kindly tests both positive and negative scenarios of the can_use_services? method as previous code reviwer seggested 👍